caluma/tests/snapshots/snap_test_schema.py

```diff
@@ -361,8 +381,8 @@
 
 interface Question {
   id: ID!
-  created: DateTime!
-  modified: DateTime!
+  createdAt: DateTime!
+  modifiedAt: DateTime!
```

Comment:
Where is `createdByUser` and `createdByGroup`?

Reply:
added here as well.